content/setup/_index.md

```diff
@@ -143,6 +142,7 @@ server {
   ssl_stapling on;
   ssl_stapling_verify on;
 
+	# location of the build of file-picker
```

Comment:
I think indentation here should be fixed to use space like the rest of the document